packages/shorebird_cli/lib/src/commands/preview_command.dart

```diff
@@ -308,4 +341,84 @@ class PreviewCommand extends ShorebirdCommand {
       '${platform.name}_${release.version}.$extension',
     );
   }
+
+  Future<void> setChannelOnRunner({
```

Comment:
This could use a doc comment explaining that it's finding the shorebird.yaml embedded in the `Runner.app` and setting the `channel` field.